addons/website/views/snippets/snippets.xml

```diff
@@ -111,7 +111,6 @@
                     <t id="sale_products_hook"/>
                     <t t-snippet="website.s_searchbar" t-thumbnail="/website/static/src/img/snippets_thumbs/s_searchbar.svg"/>
                     <t id="blog_posts_hook"/>
-                    <t id="event_local_events_hook"/>
```

Comment:
To answer what this commit says in its commit message: "Be careful that it's used in the customize menu so we'll have
to change the target to the new snippet ?:

If we follow our website migration logic: either there is a migration script with this PR to convert old user snippet in pages & fields to the new snippet, or the new snippet should be a new version of that old snippet (with a 001.js replacing an old (but kept) 000.js) etc. Of course, always better to migrate when possible instead. That also applies to the customize menu option.